playground/package.json

```diff
@@ -2,11 +2,14 @@
   "name": "cooklang-playground",
   "version": "0.0.0",
   "license": "MIT",
+  "private": true,
   "scripts": {
     "start": "vite",
-    "watch": "vite",
     "build": "vite build"
   },
+  "dependencies": {
+    "@cooklang/cooklang-ts": "*"
+  },
```

Comment:
This should still work fine with the GitHub pages version, but also allows whatever version for the stackblitz template.